crates/primitives/src/chain/mod.rs

```diff
@@ -44,6 +47,18 @@ impl Chain {
         Chain::Named(ethers_core::types::Chain::Sepolia)
     }
 
+    /// Returns the optimism goerli chain.
+    #[cfg(feature = "optimism")]
+    pub const fn optimism_goerli() -> Self {
+        Chain::Named(ethers_core::types::Chain::OptimismGoerli)
+    }
+
+    /// Returns the optimism mainnet chain.
+    #[cfg(feature = "optimism")]
+    pub const fn optimism_mainnet() -> Self {
+        Chain::Named(ethers_core::types::Chain::Optimism)
```

Comment:
Does ethers_core have these chains by default or we need to hoist as a feature dependency?

Reply:
They're included in the default features; Didn't have to add a feature flag to get them.